src/Nest/Mapping/AttributeBased/ElasticsearchCorePropertyAttributeBase.cs

```diff
@@ -2,7 +2,7 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
-﻿using System;
+ using System;
```

Comment:
@Mpdreamz, Rider is automatically adding this space in when a file is opened. It looks like it's related to the introduction of the license header.